ray-on-gke/tpu/kuberay-tpu-webhook/webhook_main_test.go

```diff
@@ -1126,33 +953,31 @@ func Test_CheckWorkersMatchTopology(t *testing.T) {
 }
 
 func Test_ValidateRayCluster(t *testing.T) {
```

Comment:
Can we add a test where with multiple worker groups, where 1 worker group requests TPUs and the other requests GPUs?

Reply:
Done in [78e15a8](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/723/commits/78e15a864ce8797d6c3ae2f3acbf0f2c2ba26751).